File: backend/src/main/java/com/interview/platform/mod14_ai_evaluation/repository/AiEvaluationRepository.java
package com.interview.platform.mod14_ai_evaluation.repository;

import com.interview.platform.mod14_ai_evaluation.model.AiEvaluationEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import java.util.UUID;

public interface AiEvaluationRepository extends JpaRepository<AiEvaluationEntity, UUID> {
}

File: android-app/app/src/main/java/com/interview/platform/domain/repository/AiEvaluationRepository.kt
package com.interview.platform.domain.repository

import com.interview.platform.domain.model.mod14_ai_evaluation.AiEvaluation
import com.interview.platform.data.model.Result
import kotlinx.coroutines.flow.Flow

interface AiEvaluationRepository {
    fun fetchAll(): Flow<Result<List<AiEvaluation>>>
    suspend fun syncData(): Result<Unit>
}
